### PR TITLE
add strategy filter

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-libs/pull/11
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/71
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/39
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1019
https://github.com/open-orchestra/open-orchestra-base-api-mongo-model-bundle/pull/6
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/406
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/49

### CODE REVIEW

File: BaseApiBundle/DependencyInjection/OpenOrchestraBaseApiExtension.php [hunk 1]
```diff
@@ -37,5 +37,11 @@ public function load(array $config, ContainerBuilder $container)
         $loader->load('transformer.yml');
         $loader->load('oauth2.yml');
         $loader->load('security.yml');
+
+        $filterPagincationConfig = __DIR__.'/../../../../open-orchestra-libs/OpenOrchestra/Pagination';
```
Comment (nicolasThal) [inline BaseApiBundle/DependencyInjection/OpenOrchestraBaseApiExtension.php]: why did you put that into the base-api-bundle when you should have put it in the base-bundle

Comment (nicolasThal) [inline BaseApiBundle/DependencyInjection/OpenOrchestraBaseApiExtension.php]: the directory and file names should be as configuration parameter also

**Comment by alavieille** [comment]
@nicolasThal

File: BaseApiBundle/OpenOrchestraBaseApiBundle.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace OpenOrchestra\BaseApiBundle;
 
+use OpenOrchestra\BaseApiBundle\DependencyInjection\Compiler\FilterTypePaginationCompilerPass;
```
Comment (nicolasThal) [inline BaseApiBundle/OpenOrchestraBaseApiBundle.php]: you can suppress this use

**Comment by alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**